src/Parsing/ParserState.php

```diff
@@ -520,7 +520,7 @@ private function strsplit($sString)
             if ($sString === '') {
                 return [];
             } else {
-                return str_split($sString);
+                return mb_str_split($sString);
```

Comment:
The check in line 520 ensures that this `str_split` call will only happen with a none-empty string. So I don't understand why the behavior with an empty string would be a problem here. Am I missing something?

Reply:
Yes, you're correct @oliverklee.
Sorry for the noise.
I'm going to close this PR

Thanks,